feature/mpls/otg_tests/static_bgp_nexthop/README.md

```diff
@@ -0,0 +1,118 @@
+4)  Enable MPLS forwarding.
+5)  Create egress static LSP for IPv4 and IPV6 traffic to pop the label and resolve the next-hop BGP-NH-V4 and BGP-NH-V6 respectivelly
+
+```yaml
+network-instances:  
+  - network-instance:  
+    mpls:  
+      lsps:  
+        static-lsps:  
+          - static-lsp:  
+            config:  
+              name: "lsp-egress-v4"  
+            egress:  
+              next-hop: 203.0.200.1 
+              incoming-label: 1000004  
+          - static-lsp:  
+            config:  
+              name: "lsp-egress-v6"  
+            egress:  
+              next-hop: 2001:db8:128:200::1  
+              incoming-label: 1000006
+```
+    *  Set resolve NH action for both LSPs.
+
+**TODO:** OC model does not support resolve next-hop option for LSPs.
+
+7)  Configure static routes i.e. `IPV4-DST = 203.0.113.0/24` and `IPV6-DST = 2001:db8:128:128::/64` to ATE Port 3.
+```yaml
+network-instances:
+  - network-instance:
+    protocols:
+      - protocol:
+        static-routes:
+          - static:
+            config:
+              prefix: "203.0.113.0/24"
+            next-hops:
+              - next-hop:
+                config:
+                  index:  1
+                  next-hop: "ATE PORT 3"
+          - static:
+            config:
+              prefix: "2001:db8:128:128::/64"
+            next-hops:
+              - next-hop:
+                config:
+                  index:  1
+                  next-hop: "ATE PORT 3"
+```
+
+### MPLS-2.2.1: Verify IPv4 MPLS forwarding
+
+*   Push the above DUT configuration.
+*   Start traffic flow with MPLS[lbl-1000004] and IPv4 destined to IPV4-DST.
+*   Verify that traffic arrives to ATE Port 2.
+
+### MPLS-2.2.2: Verify IPv6 MPLS forwarding
+
+*   Push the above DUT configuration.
+*   Start traffic flow with MPLS[lbl-1000006] and IPv4 destined to IPV6-DST.
+*   Verify that traffic arrives to ATE Port 2.
+
+### MPLS-2.2.3: Verify IPv4 traffic discard when BGP-NH is not available.
+
+*   Withdraw BGP-NH-V4 advertisement.    
+*   Push the above DUT configuration.
+*   Start traffic flow with MPLS[lbl-1000004] and IPv4 destination set to IPV4-DST.
+*   Verify that traffic is discarded.
+
+### MPLS-2.2.4: Verify IPv6 traffic discard when BGP-NH is not available.
+
+*   Withdraw BGP-NH-V6 advertisement.    
+*   Push the above DUT configuration.
+*   Start traffic flow with MPLS[lbl-1000006] and IPv6 destination set to IPV6-DST.
+*   Verify that traffic is discarded.
+
+## OpenConfig Path and RPC Coverage
+
+```yaml
+paths:
+  ## Config paths
+  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/next-hop:
+  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/incoming-label:
+
```

Comment:
```suggestion
  ## Config paths
  /network-instances/network-instance/protocols/protocol/static-routes/config/prefix
  /network-instances/network-instance/protocols/protocol/static-routes/config/index:
  /network-instances/network-instance/protocols/protocol/static-routes/next-hops/next-hop/config/next-hop:

  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/name: 
  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/next-hop:
  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/incoming-label:
```

Reply:
Hey Daren,

Those paths are not invalid OCPaths

Reply:
Sorry about that!  Note,  in future you can find and correct these yourself by searching at https://openconfig.net/projects/models/paths/

Here's my suggestion though:

```
/network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/incoming-label 
/network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/next-hop 

/network-instances/network-instance/protocols/protocol/static-routes/static/config/prefix 
/network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/next-hop
/network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/index
```